NetCore/project.json

```diff
@@ -5,18 +5,19 @@
     "emitEntryPoint": true
   },
   "frameworks": {
-    "netcoreapp1.0": {
+    "netcoreapp1.1": {
```

Comment:
Fair enough. I've faced issues with Benchmark.NET as well, which I've mostly solved by picking the latest build that works with 1.0, but it may be time to relax that and go to 1.1. OK, well, never mind then, I'll just merge this as is. Thanks again.